powerhome/playbook
# [Play-254] ♻️ Typescript Conversion: Filter

#### Screens

<img width="1424" alt="Screen Shot 2023-02-27 at 11 19 38 AM" src="https://user-images.githubusercontent.com/73671109/221619653-1a3f8c1d-7377-4259-ba83-e1447adcdd29.png">

#### Breaking Changes

NO

#### Runway Ticket URL

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-254)

#### How to test this

Check filter kit usage in Nitro and ensure all is working as expected after typescript converison

#### Checklist:
